docs/components/popover.md

```diff
@@ -262,6 +262,17 @@ You can pass in an optional flag to not show the caret on the edge of the popove
 </KPop>
 ```
 
+### onAlreadyOpen
+You can pass in an optional callback function to trigger when the popup is already open (usually when the trigger method is by clicking).
+This can be useful in situations where for example, you need to close the popup.
+
+```vue
+<KPop title="Cool header" on-already-open="closePopup()">
```

Comment:
what is `closePopup`? It seems like we are making the user think too much about implementation. Why not have a prop called `is-open` that is reactive and then the user can decide when they want the popover opened or not? This is one of the problems I've had with vue-autosuggest, so we implemented `should-render-suggestions` which allows users to hook into the state of the autosuggest and close it based on whatever criteria they select, but they can also see how `vue-autosuggest` decides to close it. https://github.com/darrenjennings/vue-autosuggest#shouldrendersuggestions 
I think users generally prefer a reactive `is-open` prop because it's straightforward.

Reply:
I don't have the actual implementation for `closePopup` the same reason why I don't have it for `zoom` in the docs is because its left up to the user, an example component might be too complex for the docs.

The reason why it's a callback and not a boolean is explained here: https://github.com/Kong/khcp/pull/561#issuecomment-704757189

Reply:
yea but as the docs currently stand, it's not apparent how a user can control the open/close state of kpop. i.e. closePopup should not be a user implemented function.

Reply:
what do you recommend then? `closePopup should not be a user implemented function.`  I think you are misunderstanding the use of this prop. Because this prop isn't meant to be a close popup function all the time, it's a custom callback function. Using `hidePopper` is just one way the user can do it, but they can choose to do something else when the popup is already opened, like toggling something. I can put in a simple example.

Reply:
the el.hidePopper is the part that is what I want to avoid. `kpopRef.hidePopper` is an escape hatch that is subject to change based on the implementation of kpop.

Reply:
yeah, but this prop doesn't have to be used for closing the popup. That's just how we choose to use it in the Dropdown. Since Dropdown has no visibility into the click handler implementation of Kpop, I dont see a better way of doing it than using a callback right now. I'm on all ears for a better solution but I spent 3 hours up to midnight last night racking my brain on how to do this and this is was the best solution I could find that was as general of a fix as possible for our dropdowns.